azkustoingest/file_options.go

```diff
@@ -229,16 +229,18 @@ func InferFormatFromFileName(fName string) DataFormat {
 // IngestionMapping provides runtime mapping of the data being imported to the fields in the table.
 // "ref" will be JSON encoded, so it can be any type that can be JSON marshalled. If you pass a string
```

Comment:
```suggestion
// IngestionMapping provides runtime mapping of the data being imported to the columns in the table.
// "mapping" will be JSON encoded, so it can be any type that can be JSON marshalled. If you pass a string
```